playkit/src/main/java/com/kaltura/playkit/PlayerEvent.java

```diff
@@ -113,10 +113,12 @@ public MetadataAvailable(List<PKMetadata> metadataList) {
     public static class SourceSelected extends PlayerEvent {
 
         public final String sourceUrl;
+        public final String sourceExtention;
```

Comment:
I needed it for reporting 
the protocol hls/mpegdash/http
right now it was hard coded hls

anyway I will pass now the whole source object